api/src/cctp_calls.rs

```diff
@@ -805,6 +870,29 @@ mod test {
         assert!(vrf_proof_to_hash(&wrong_msg, &pk, &vrf_proof).is_err());
     }
 
+    #[serial]
+    #[test]
+    fn sample_calls_vrf_derive_from_seed() {
+        let (pk, sk) = vrf_derive_key_from_seed(&[1u8; 32]); //Keygen
+        assert_eq!(vrf_get_public_key(&sk), pk); //Get pk
+        assert!(vrf_verify_public_key(&pk)); //Verify pk
+
+        let (pk, sk) = vrf_derive_key_from_seed(&[]); //Keygen
+        assert_eq!(vrf_get_public_key(&sk), pk); //Get pk
+        assert!(vrf_verify_public_key(&pk)); //Verify pk
+    }
+
+    #[serial]
+    #[test]
+    fn sample_calls_schnorr_vrf_derive_different_keys_from_same_seed() {
+        const SEED: [u8; 32] = [7u8; 32];
+        let (spk, ssk) = schnorr_derive_key_from_seed(&SEED); //Keygen
+        let (vpk, vsk) = vrf_derive_key_from_seed(&SEED); //Keygen
+
+        assert_ne!(vpk, spk);
+        assert_ne!(vsk, ssk);
+    }
```

Comment:
Let's add some regression tests for both schnorr and vrf keys as it was done in the Java part.
It seems more safe and useful, considering that only rust part `sc-sctyprolib` can be used by third parties. So they will be sure that the algorithm remains the same.

Reply:
Added.